src/components/Drawer.js

```diff
@@ -227,7 +227,7 @@ class Drawer extends React.Component {
     const titleUniqueId = _uniqueId('mx-drawer-title-');
 
     return (
-      <StyleRoot>
+      <div>
```

Comment:
Do we still need this div?

Reply:
Nope, I should have removed that. Just pushed that change.